spec/collator.html

```diff
@@ -35,6 +35,10 @@ <h1>InitializeCollator ( _collator_, _locales_, _options_ )</h1>
         1. Set _opt_.[[kn]] to _numeric_.
         1. Let _caseFirst_ be ? GetOption(_options_, *"caseFirst"*, *"string"*, &laquo; *"upper"*, *"lower"*, *"false"* &raquo;, *undefined*).
         1. Set _opt_.[[kf]] to _caseFirst_.
+        1. Let _collation_ be ? GetOption(_options_, `"collation"`, `"string"`, *undefined*, *undefined*).
+        1. If _collation_ is not *undefined*, then
+          1. If _collation_ does not match the Unicode Locale Identifier `type` nonterminal, throw a *RangeError* exception.
```

Comment:
#175 also says:

```
1. Set _calendar_ to the result of mapping _calendar_ to lower case as as specified in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>.
```

Reply:
No, we don't need that now. See https://github.com/tc39/ecma402/commit/ca92351087c1a66631f86900c1dc3c2d1ef30f10#diff-1f2e1333fe08618e8af2d7344b031dc1